examples/tutorials/advanced/SMPC_Protocols_Explanation.ipynb

```diff
@@ -0,0 +1,1112 @@
+{
```

Comment:
<p>(the first category) &lt;-- also a dot here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done.</p>